pypowerwall/tedapi/vitals_dictionary.py

```diff
@@ -0,0 +1,517 @@
+import json
```

Comment:
"vitals_dictionary" is too verbose, you could just call it "vitals.py" -- It's already under TEDAPI, that's self-documenting enough.